MPF.ExecutionContexts/Redumper/ExecutionContext.cs

```diff
@@ -393,13 +395,16 @@ protected override bool ValidateAndSetParameters(string? parameters)
                     case CommandStrings.Hash:
                     case CommandStrings.Info:
                     case CommandStrings.Skeleton:
-                    //case CommandStrings.FlashMT1339:
+                    case CommandStrings.FlashMT1339:
                     case CommandStrings.Subchannel:
                     case CommandStrings.Debug:
                     case CommandStrings.FixMSF:
                     case CommandStrings.DebugFlip:
                     case CommandStrings.DriveTest:
-                        ModeValues.Add(part);
+                        // Only allow one mode per command
+                        if (ModeValue != null)
+                            return false;
```

Comment:
I think it's reasonable to just discard any additional mode values provided. An alternative is overwriting.